vowpalwabbit/constant.h

```diff
@@ -31,7 +31,7 @@ constexpr unsigned char spelling_namespace = 133;      // this is \x85
 constexpr unsigned char conditioning_namespace = 134;  // this is \x86
 constexpr unsigned char dictionary_namespace = 135;    // this is \x87
 constexpr unsigned char node_id_namespace = 136;       // this is \x88
-constexpr unsigned char message_namespace = 137;       // this is \x89
+constexpr unsigned char baseline_enabled_message_namespace = 137;  // this is \x89
```

Comment:
was message_namespace used for baseline.cc before? or is this being reused?

Reply:
It was only used by baseline.cc. I am making it clearer what it does